packages/jest-puppeteer-axe/src/index.js

```diff
@@ -53,17 +53,19 @@ function formatViolations( violations ) {
  *
  * @see https://github.com/dequelabs/axe-puppeteer
  *
- * @param {puppeteer.Page} page                 Puppeteer's page instance.
- * @param {?Object}        params               Optional Axe API options.
- * @param {?string|Array}  params.include       CSS selector(s) to add to the list of elements
- *                                              to include in analysis.
- * @param {?string|Array}  params.exclude       CSS selector(s) to add to the list of elements
- *                                              to exclude from analysis.
- * @param {?Array}         params.disabledRules The list of Axe rules to skip from verification.
+ * @param {puppeteer.Page}  page                 Puppeteer's page instance.
+ * @param {?Object}         params               Optional Axe API options.
+ * @param {?string|Array}   params.include       CSS selector(s) to add to the list of elements
+ *                                               to include in analysis.
+ * @param {?string|Array}   params.exclude       CSS selector(s) to add to the list of elements
+ *                                               to exclude from analysis.
+ * @param {?Array}          params.disabledRules The list of Axe rules to skip from verification.
+ * @param {?Axe.RunOptions} params.options       Options to be used by axe.run.
```

Comment:
It would be great to provide more detailed info about what kind of options are allowed and how they differ from `include`, `exclude` or `disabledRules`.

Reply:
Sorry, I'm not sure what you mean here. Should I add this link in the jsdoc? https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter

Reply:
Should we update this part?
```js
 * @param {?Object}         params               Optional Axe API options.
```

and this is how they document it in official docs:

> The options parameter is flexible way to configure how `axe.run` operates.

so we probably could hide `axe.run` bit with:

> Options is a flexible way to configure how Axe run operates.

Reply:
Can you please suggest some new wording  for `params`?